src/styles/game.css

```
@@ -0,0 +1,609 @@
+@import url("https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap");
```

Stylelint (17.2.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap"" (import-notation)

(import-notation)